pom.xml

```diff
@@ -53,18 +53,18 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!--BOMs-->
-        <spring-cloud-bom.version>Hoxton.SR2</spring-cloud-bom.version>
-        <spring-boot-dependencies-bom.version>2.2.4.RELEASE</spring-boot-dependencies-bom.version>
+        <spring-cloud-bom.version>Hoxton.SR3</spring-cloud-bom.version>
+        <spring-boot-dependencies-bom.version>2.2.5.RELEASE</spring-boot-dependencies-bom.version>
 
         <!--other-->
         <json-reactor.version>0.0.1</json-reactor.version>
 
         <!-- Tests -->
-        <wiremock.version>2.26.0</wiremock.version>
+        <wiremock.version>2.26.3</wiremock.version>
         <awaitility.version>4.0.2</awaitility.version>
-        <guava.version>23.0</guava.version>
+        <guava.version>28.2-android</guava.version>
```

Comment:
why not ```28.2-jre```?

Reply:
From what I understand it is slightly different. Should this also work on android, if so, then feel that is likely safer choice?

Reply:
You may try to remove this dependency at all. I don't see any usage of it as for now.

Reply:
Done